pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -292,6 +324,18 @@ func (r *reconciler) createLoadBalancerService(service *corev1.Service) error {
 	return nil
 }
 
+// deleteLoadBalancerService deletes a load balancer service.
+func (r *reconciler) deleteLoadBalancerService(service *corev1.Service, options *crclient.DeleteOptions) error {
+	if err := r.client.Delete(context.TODO(), service, options); err != nil {
```

Comment:
Is there a foreseeable reason to include an `*crclient.DeleteOptions` parameter? I'm not opposed to adding one here, but I recognize that the other `r.client.Delete` calls in this repo exclude this paramter.

Reply:
Good question, and I had to go back to see why I did this.  Yes, we will want to specify `PropagationPolicy: "Foreground"` when deleting and recreating the service if/when we re-introduce mutable scope.

Reply:
aha! That makes sense. Looking forward to taking advantage of that :wink: